### DISCUSSION TITLE
[Request] Kopia

### DISCUSSION THREAD

**Discussion by c0bnut** [tteck/Proxmox#343] [Ideas]
Cross-platform backup tool for Windows, macOS & Linux with fast, incremental backups, client-side end-to-end encryption, compression and data deduplication. CLI and GUI included. Specifically I would like to request KopiaUI in server mode.

https://kopia.io/
https://github.com/kopia/kopia

**Comment by tteck** [comment]
I still don't see the value of a tool such as this for Proxmox. Maybe you can explain the value over the existing backup schema Proxmox offers.

**Reply by c0bnut** [reply]
Proxmox only allows a full backup of VM's and Containers only. I already have this backup set to go to a ZFS pool with mechanical disks. I want to then backup the Proxmox backups as well as other general data on the pool to an external hard drive as well as a cloud service with compression, deduplication, encryption and incremental abilities. That way I have a 3-2-1 backup solution. A GUI would make this easier and I could add it to my dashboard as a link for remote management.

**Reply by tteck** [reply]
Can't rclone do all of that?

**Reply by c0bnut** [reply]
Potentially, however rclone lists compression and their web GUI as experimental and not for production use.

**Reply by lucasjose501** [reply]
Can kopia be used as server endpoint for my computers? So I can backup other computers and smartphones over network to the server and the server sends it to the cloud

Edit: reading about it I see that is better to use something like nextcloud to sync the devices and kopia to back it up to the cloud

**Reply by c0bnut** [reply]
You could use rclone to backup a Windows or Linux machine to your server or even the built in 'Backup & Restore' application in Windows and NextCloud for the smartphones. I don't believe NextCloud backs up things like SMS or Call logs, so you might be better off using Titanium Backup in that case.
